app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentConvertDialog.java

```diff
@@ -0,0 +1,39 @@
+package com.mithridat.nonoconverter.ui.editimage;
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+
+import com.mithridat.nonoconverter.R;
+
+import androidx.fragment.app.DialogFragment;
+import androidx.fragment.app.FragmentActivity;
+
+public class FragmentConvertDialog extends DialogFragment{
```

Comment:
It looks like `package-private`

Reply:
Android studion said that it must be public here